seed/analysis_pipelines/better/pipeline.py

```diff
@@ -351,10 +356,10 @@ def _start_analysis(self, analysis_id):
             context,
         )
 
-    _store_better_building_analysis_results(
-        better_building_analyses,
-        context,
-    )
+        _store_better_building_analysis_results(
+            better_building_analyses,
+            context,
+        )
```

Comment:
`_strore_better_building_analysis_results` should only be run if `better_portfolio_id` is None